File: solutions/go/01/mod/main.go
package main

import "fmt"

const mod = 998244353

func addMod(a, b, m int) int {
	return (a + b) % m
}

func main() {
	fmt.Println(addMod(1_000_000_000, 1_000_000_000, mod))
}

File: solutions/python/01/mod/main.py
MOD = 998244353


def add_mod(a: int, b: int, mod: int = MOD) -> int:
    return (a + b) % mod


def mul_mod(a: int, b: int, mod: int = MOD) -> int:
    return (a * b) % mod


def main() -> None:
    print(add_mod(10**9, 10**9))
    print(mul_mod(10**5, 10**5))


if __name__ == "__main__":
    main()
